src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -0,0 +1,197 @@
+  @Prop() public accessibilityLabelledby: string | undefined;
+
+  /** Event emitted when the value of the inner HTMLInputElement changes. */
+  @Event() public sbbChange: EventEmitter<SbbSliderChange>;
+
+  /**
+   * The ratio between the absolute value and the validity interval.
+   * E.g. given `min=0`, `max=100` and `value=50`, then `_valueFraction=0.5`
+   */
+  @State() private _valueFraction = 0;
+
+  /** Host element */
+  @Element() private _element!: HTMLElement;
+
+  /** Reference to the inner HTMLInputElement with type='range'. */
+  private _rangeInput!: HTMLInputElement;
+
+  /** Indicates if the component is used within a sbb-form-field. */
+  private _isInFormField: boolean;
+
+  public connectedCallback(): void {
+    this._isInFormField = !!hostContext('sbb-form-field', this._element);
+    this._handleChange();
+  }
+
+  /**
+   * Recalculates the `_valueFraction` on change to correctly display the slider knob and lines.
+   * The first calculation happens in connectedCallback(...), so since `_rangeInput` is not yet available,
+   * the `min` and `max` values are used; if `value` is not provided, the default value is halfway between min and max
+   * (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#value).
+   */
+  private _handleChange(): void {
+    let min: number, max: number, value: number;
+    if (this._rangeInput) {
+      min = +this._rangeInput.min;
+      max = +this._rangeInput.max;
+      value = this._rangeInput.valueAsNumber;
+    } else {
+      min = +this.min;
+      max = +this.max;
+      value =
+        this.value && this.value !== ''
+          ? +this.value
+          : this.valueAsNumber
+          ? this.valueAsNumber
+          : +this.min + (+this.max - +this.min) / 2;
+    }
+    const mathFraction: number = (value - min) / (max - min);
+    this._valueFraction =
+      isNaN(mathFraction) || mathFraction < 0 ? 0 : mathFraction > 1 ? 1 : mathFraction;
+  }
+
+  /** Emits the change event. */
+  private _emitChange(): void {
+    this.sbbChange.emit({
+      value: this._rangeInput.valueAsNumber,
+      max: +this._rangeInput.max,
+      min: +this._rangeInput.min,
+    });
+  }
+
+  public render(): JSX.Element {
+    const inputAttributes = {
+      name: this.name || null,
+      form: this.form || null,
+      min: this.min || null,
+      max: this.max || null,
+      disabled: this.disabled || this.readonly || null,
+      valueAsNumber: this.valueAsNumber || null,
+      value: this.value || null,
+      'aria-label': this.accessibilityLabel || null,
+      'aria-describedby': this.accessibilityDescribedby || null,
+      'aria-labelledby': this.accessibilityLabelledby || null,
+    };
+
+    return (
+      <Host class={{ 'sbb-form-field-element': this._isInFormField }}>
+        <div class="sbb-slider__wrapper">
+          {this.startIcon && (
+            <slot name="prefix">
+              <sbb-icon name={this.startIcon} />
+            </slot>
+          )}
```

Comment:
The slot should always be rendered.
```suggestion
          <slot name="prefix">
            {this.startIcon && <sbb-icon name={this.startIcon} />}
          </slot>
```